ui/perfherder/alerts/AlertTable.jsx

```diff
@@ -217,32 +250,29 @@ export default class AlertTable extends React.Component {
                     xs={10}
                     className="text-left alert-summary-header-element"
                   >
-                    <FormGroup check>
-                      <Label check className="pl-1">
-                        <Input
-                          data-testid={`alert summary ${alertSummary.id.toString()} checkbox`}
-                          aria-labelledby={`alert summary ${alertSummary.id.toString()} title`}
-                          type="checkbox"
-                          checked={allSelected}
-                          disabled={!user.isStaff}
-                          onChange={() =>
-                            this.setState({
-                              allSelected: !allSelected,
-                              selectedAlerts: !allSelected
-                                ? [...alertSummary.alerts]
-                                : [],
-                            })
-                          }
-                        />
-                        <AlertHeader
-                          frameworks={frameworks}
-                          alertSummary={alertSummary}
-                          repoModel={repoModel}
-                          issueTrackers={issueTrackers}
-                          user={user}
-                          updateAssignee={this.updateAssignee}
-                        />
-                      </Label>
+                    <FormGroup check className="d-inline-flex">
+                      <SelectAlertsDropdown
+                        selectAlertsByStatus={this.selectAlertsByStatus}
+                        toggleAllAlerts={() => {
```

Comment:
Couldn't we remove this small helper function? We seem to already do the same thing via `selectAlertByStatus(all)`.